Hymns/Display/SoundCloudWebView.swift

```diff
@@ -0,0 +1,56 @@
+import SwiftUI
+import WebKit
+
+struct SoundCloudWebView: UIViewRepresentable {
+    var url: URL
+    var loadStatusChanged: ((Bool, Error?) -> Void)?
+
+    func makeCoordinator() -> SoundCloudWebView.Coordinator {
+        Coordinator(self)
+    }
+
+    func makeUIView(context: Context) -> WKWebView {
+        let view = WKWebView()
+        view.navigationDelegate = context.coordinator
+        view.load(URLRequest(url: url))
+        return view
+    }
+
+    func updateUIView(_ uiView: WKWebView, context: Context) {
+    }
+
+    func onLoadStatusChanged(perform: ((Bool, Error?) -> Void)?) -> some View {
+        var copy = self
+        copy.loadStatusChanged = perform
```

Comment:
@finben still unsure what this could. Could you help me clarify why we have these?

Reply:
They are used to get the delegate and coordinator working. Without them we can't use the coordinator correctly to inject javascript into the page. https://developer.apple.com/forums/thread/126986